static/script-tests/tests/devices/mediaplayer/cehtmlcommontests.js

```diff
@@ -1369,6 +1369,46 @@ window.commonTests.mediaPlayer.cehtml.mixinTests = function (testCase, mediaPlay
         });
     };
 
+    mixins.testGetDurationReturnsInfinityWithALiveVideoStream = function(queue) {
```

Comment:
on further investigation the runMediaPlayerTest() function has differences for html5, cehtml and samsung_maple. need to analyse if they could be merged.